src/Aura/Languages.hs

```diff
@@ -1576,7 +1580,7 @@ singleSync_1 (bt -> p) = \case
 absInfoFields :: Language -> [String]
 absInfoFields = \case
     Croatian   -> [ "Repozitorij","Ime","Verzija","Zavisnosti","Make Zavisnosti","Opis" ]
-    German     -> [ "Quelle","Name","Version","Hängt ab von","Make Abh.","Beschreibung"]
+    German     -> [ "Repository","Name","Version","Hängt ab von","Make-Abhängigkeiten","Beschreibung"]
```

Comment:
'Quelle' might be a usable translation, but it's rather unclear (would be a direct translation for 'source') and I think 'Repository' is a widely known term among german linux sys-admins.